Mathlib/Data/List/Perm/Basic.lean

```diff
@@ -37,7 +37,7 @@ lemma perm_rfl : l ~ l := Perm.refl _
 attribute [symm] Perm.symm
 attribute [trans] Perm.trans
 
-instance : IsSymm (List α) Perm := ⟨fun _ _ ↦ .symm⟩
+instance : Std.Symm <| @Perm α := ⟨fun _ _ ↦ .symm⟩
```

Comment:
nit but this feels like an overkill usage for <|
```suggestion
instance : Std.Symm (@Perm α) := ⟨fun _ _ ↦ .symm⟩
```

Reply:
I thought style choices were up to the author